### PR TITLE
Changes to MaxRequestBodySize comment

### PR DESCRIPTION
**Pull request by andyo-tyk**
Changes following review of DX-311

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)

## Checklist

- [ ] I ensured that the documentation is up to date
- [ ] I explained why this PR updates go.mod in detail with reasoning why it's required
- [ ] I would like a code coverage CI quality gate exception and have explained why

### CODE REVIEW

**Comment by andyo-tyk** [comment]
This relates to changes in TT-5186

**Review by lghiur — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: cancelled** :no_entry_sign: 
Branch used: refs/pull/5071/merge
Commit:  
Triggered by: pull_request (@andyo-tyk)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5132849031)

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign: 
Branch used: refs/pull/5071/merge
Commit: 5fa8058f5099e54758353e07f56913f181f2fe16 
Triggered by: pull_request (@andyo-tyk)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5199460132)

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Apologies Andy, I forgot to comment the suggestion string text for `MaxRequestBodySize` in config.go.

I have added suggestion for commit

File: config/config.go [hunk 1]
```diff
@@ -410,13 +410,15 @@ type HttpServerOptionsConfig struct {
 	// Custom SSL ciphers. See list of ciphers here https://tyk.io/docs/basic-config-and-security/security/tls-and-ssl/#specify-tls-cipher-suites-for-tyk-gateway--tyk-dashboard
 	Ciphers []string `json:"ssl_ciphers"`
 
-	// MaxRequestBodySize configures the maximum request body size in bytes.
+// MaxRequestBodySize configures a maximum size limit for request body size (in bytes) for all APIs on the Gateway.
```
Comment (dcs3spp) [inline config/config.go:413]:
```suggestion
	// MaxRequestBodySize configures a maximum size limit for request body size (in bytes) for all APIs on the Gateway.
```

**Review by titpetric — APPROVED** [review]
LGTM

**Review by dcs3spp — APPROVED** [review]
lgtm

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign: 
Branch used: refs/pull/5071/merge
Commit: 8b3394caa0b64ead1ab06d5fd354bc28340ca510 
Triggered by: pull_request (@jeffy-mathew)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5209831026)

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign: 
Branch used: refs/pull/5071/merge
Commit: 8b3394caa0b64ead1ab06d5fd354bc28340ca510 
Triggered by: pull_request (@jeffy-mathew)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5209831026)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5071/merge
Commit: 8b3394caa0b64ead1ab06d5fd354bc28340ca510 
Triggered by: pull_request (@jeffy-mathew)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5209831026)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5071/merge
Commit: 406d44db3cea4ddf585ead15150497e6dc9fc2e4 
Triggered by: pull_request (@lghiur)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5298101521)

**Pull request merged**